cmd/func/main.go

```diff
@@ -38,7 +39,9 @@ func main() {
 		}}
 
 	if err := cmd.NewRootCmd(cfg).ExecuteContext(ctx); err != nil {
-		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		if !errors.Is(err, terminal.InterruptErr) {
+			fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		}
 		if ctx.Err() != nil {
 			os.Exit(130)
```

Comment:
```suggestion
		}
		if ctx.Err() != nil || errors.Is(err, terminal.InterruptErr) {
			os.Exit(130)
```

Reply:
Maybe I am mis-reading this... but the `|| errors.Is(err, terminal.InterruptErr)` can never be true if err is nil (which is caught by the first part of the conditional).